src/pages/kimdongki/Login/KimLogin.js

```diff
@@ -1,7 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './KimLogin.scss';
 
 function KimLogin() {
-  return <div>김동기 로그인페이지</div>;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/KimMain');
+  };
+
+  const [id, idValue] = useState('');
+  const [pw, pwValue] = useState('');
+
+  const saveUserId = e => {
+    idValue(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    pwValue(e.target.value);
+  };
+
+  const loginValid = id && id.includes('@') && pw && pw.length >= 5;
+
+  const pressEnter = e => {
+    if (e.code === 'Enter') {
+      goToMain();
+    }
+  };
+
+  return (
+    <main>
+      <div className="login_wrapper">
+        <div className="login_logo">
+          <h1>Westagram</h1>
+        </div>
+        <div className="login_container">
+          <input
+            onChange={saveUserId}
+            type="text"
+            className="login_email"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+          />
+          <input
+            onChange={saveUserPw}
+            onKeyDown={pressEnter}
+            type="password"
+            className="login_pw"
+            placeholder="비밀번호"
+          />
+
+          <button
+            className="login_btn"
+            // type="button"
+            onClick={goToMain}
+            disabled={!loginValid ? true : false}
+          >
+            로그인
+          </button>
+        </div>
+        <span className="passwordfind_link">
+          <a href="">비밀번호를 잊으셨나요?</a>
```

Comment:
a링크 대신 Link 컴포넌트 사용을 추천드립니다. 링크컴포넌트를 사용하기 위해서 import를 해줘야 합니다. 그리고 href속성으로 #을 주면 링크를 클릭해도 이동하지 않습니다. 이는 링크가 준비되어있지 않을때 많이 사용합니다. 앞으로 유용하게 사용하길 기대 합니다!
```suggestion
          <Link to="#">비밀번호를 잊으셨나요?</Link>
```